tools/pipelines-tasks/test/testhelpers.ts

```diff
@@ -189,6 +193,52 @@ export const setUpUpdateAppInstallerFileArguments = (taskMockRunner: tmrm.TaskMo
     taskMockRunner.setInput('versionUpdateMethod', 'revision');
 }
 
+export const setUpAzureArmRestMock = (taskMockRunner: tmrm.TaskMockRunner): void => {
```

Comment:
Adding JSDoc comments to the functions could improve readability and maintainability.

Reply:
Done